--- base/Ytsaurus.yson
+++ test/Ytsaurus.yson
@@ -1,10 +1,10 @@
 apiVersion: cluster.ytsaurus.tech/v1
 kind: Ytsaurus
 metadata:
-  generation: 1
+  generation: 2
   name: test-ytsaurus
   namespace: ytsaurus-components
-  resourceVersion: "26"
+  resourceVersion: "31"
 spec:
   adminCredentials:
     name: test-ytsaurus-admin
@@ -801,7 +801,7 @@
   conditions:
   - lastTransitionTime: null
     message: 0.0.0-devel
-    observedGeneration: 1
+    observedGeneration: 2
     reason: Observed
     status: "True"
     type: OperatorVersion
@@ -809,19 +809,19 @@
     message: Ready 5 of 5; daemon set ih-ca is ready at 1 nodes; daemon set ih-ms
       is ready at 1 nodes; daemon set ih-strawberry is ready at 1 nodes; daemon set
       ih-ui is ready at 1 nodes; daemon set ih-yqla is ready at 1 nodes
-    observedGeneration: 1
+    observedGeneration: 2
     reason: Sync
     status: "True"
     type: ImageHeaterComplete
   - lastTransitionTime: null
     message: Image heating is complete
-    observedGeneration: 1
+    observedGeneration: 2
     reason: Ready
     status: "True"
     type: ImageHeaterReady
   - lastTransitionTime: null
     message: Ready
-    observedGeneration: 1
+    observedGeneration: 2
     reason: Ready
     status: "True"
     type: Master-2Ready
@@ -839,55 +839,49 @@
     type: defaultMasterInitJobCompleted
   - lastTransitionTime: null
     message: Ready
-    observedGeneration: 1
+    observedGeneration: 2
     reason: Ready
     status: "True"
     type: MasterReady
   - lastTransitionTime: null
     message: 'Pods are ready: 0 of 1, running 0, need 0'
-    observedGeneration: 1
+    observedGeneration: 2
     reason: Ready
     status: "True"
     type: DiscoveryReady
   - lastTransitionTime: null
     message: 'Pods are ready: 0 of 3, running 0, need 0'
-    observedGeneration: 1
+    observedGeneration: 2
     reason: Ready
     status: "True"
     type: DataNodeReady
   - lastTransitionTime: null
     message: 'Pods are ready: 0 of 1, running 0, need 0'
-    observedGeneration: 1
+    observedGeneration: 2
     reason: Ready
     status: "True"
     type: ExecNodeReady
   - lastTransitionTime: null
     message: 'Pods are ready: 0 of 1, running 0, need 0'
-    observedGeneration: 1
+    observedGeneration: 2
     reason: Ready
     status: "True"
     type: ControllerAgentReady
   - lastTransitionTime: null
     message: 'Pods are ready: 0 of 1, running 0, need 0'
-    observedGeneration: 1
+    observedGeneration: 2
     reason: Ready
     status: "True"
     type: MasterCacheReady
   - lastTransitionTime: null
     message: 'Pods are ready: 0 of 1, running 0, need 0'
-    observedGeneration: 1
-    reason: Ready
-    status: "True"
-    type: HttpProxyReady
-  - lastTransitionTime: null
-    message: 'Pods are ready: 0 of 1, running 0, need 0'
-    observedGeneration: 1
+    observedGeneration: 2
     reason: Ready
     status: "True"
     type: RpcProxyReady
   - lastTransitionTime: null
     message: Ready
-    observedGeneration: 1
+    observedGeneration: 2
     reason: Ready
     status: "True"
     type: SchedulerReady
@@ -905,7 +899,7 @@
     type: yql-agent-environmentYqlAgentInitJobCompleted
   - lastTransitionTime: null
     message: 'Pods are ready: 0 of 0, running 0, need 0'
-    observedGeneration: 1
+    observedGeneration: 2
     reason: Ready
     status: "True"
     type: UIReady
@@ -917,7 +911,7 @@
     type: userYtsaurusClientInitJobCompleted
   - lastTransitionTime: null
     message: Ready
-    observedGeneration: 1
+    observedGeneration: 2
     reason: Ready
     status: "True"
     type: YqlAgentReady
@@ -934,18 +928,6 @@
     status: "True"
     type: UpdateIsPossible
   - lastTransitionTime: null
-    message: Ready
-    observedGeneration: 1
-    reason: Ready
-    status: "True"
-    type: YtsaurusClientReady
-  - lastTransitionTime: null
-    message: Ready
-    observedGeneration: 1
-    reason: Ready
-    status: "True"
-    type: TimbertruckReady
-  - lastTransitionTime: null
     message: Job yt-strawberry-controller-init-job-cluster completed in -1ns
     observedGeneration: 1
     reason: InitJob
@@ -953,17 +935,35 @@
     type: clusterStrawberryControllerInitJobCompleted
   - lastTransitionTime: null
     message: Ready
-    observedGeneration: 1
+    observedGeneration: 2
     reason: Ready
     status: "True"
     type: StrawberryControllerReady
   - lastTransitionTime: null
+    message: 'Pods are ready: 0 of 1, running 0, need 0'
+    observedGeneration: 2
+    reason: Ready
+    status: "True"
+    type: HttpProxyReady
+  - lastTransitionTime: null
+    message: Ready
+    observedGeneration: 2
+    reason: Ready
+    status: "True"
+    type: YtsaurusClientReady
+  - lastTransitionTime: null
+    message: Ready
+    observedGeneration: 2
+    reason: Ready
+    status: "True"
+    type: TimbertruckReady
+  - lastTransitionTime: null
     message: Cluster state is Running
-    observedGeneration: 1
+    observedGeneration: 2
     reason: Running
     status: "True"
     type: ReadyToWork
-  observedGeneration: 1
+  observedGeneration: 2
   state: Running
   updateStatus: {}
 
