--- base/ConfigMap yt-discovery-config.yson
+++ test/ConfigMap yt-discovery-config.yson
@@ -80,11 +80,24 @@
                 "ds-0.discovery.ytsaurus-components.svc.cluster.local:9020";
             ];
         };
+        "override_yaml"={
+            bool=%true;
+            float=0.500000;
+            number=1;
+            string=str;
+        };
+        "override_yson"={
+            bool=%true;
+            number=1;
+            float=0.500000;
+            string=str;
+        };
     }
 metadata:
   annotations:
     ytsaurus.tech/config-hash: <censored>
-    ytsaurus.tech/observed-generation: "1"
+    ytsaurus.tech/config-overrides-version: "1"
+    ytsaurus.tech/observed-generation: "4"
   generation: 1
   labels:
     app.kubernetes.io/component: yt-discovery
@@ -102,5 +115,5 @@
     kind: Ytsaurus
     name: test-ytsaurus
     uid: ""
-  resourceVersion: "1"
+  resourceVersion: "2"
 
