--- base/ConfigMap yt-discovery-config.yson
+++ test/ConfigMap yt-discovery-config.yson
@@ -80,10 +80,12 @@
                 "ds-0.discovery.ytsaurus-components.svc.cluster.local:9020";
             ];
         };
+        hello=yt;
     }
 metadata:
   annotations:
     ytsaurus.tech/config-hash: <censored>
+    ytsaurus.tech/config-overrides-version: "1"
     ytsaurus.tech/observed-generation: "1"
   generation: 1
   labels:
@@ -102,5 +104,5 @@
     kind: Ytsaurus
     name: test-ytsaurus
     uid: ""
-  resourceVersion: "1"
+  resourceVersion: "2"
 
