--- base/Service discovery.yson
+++ test/Service discovery.yson
@@ -1,6 +1,6 @@
 metadata:
   annotations:
-    ytsaurus.tech/observed-generation: "1"
+    ytsaurus.tech/observed-generation: "4"
   generation: 1
   labels:
     app.kubernetes.io/component: yt-discovery
@@ -18,7 +18,7 @@
     kind: Ytsaurus
     name: test-ytsaurus
     uid: ""
-  resourceVersion: "1"
+  resourceVersion: "3"
 spec:
   clusterIP: None
   publishNotReadyAddresses: true
