--- base/Ytsaurus.yson
+++ test/Ytsaurus.yson
@@ -1,10 +1,10 @@
 apiVersion: cluster.ytsaurus.tech/v1
 kind: Ytsaurus
 metadata:
-  generation: 1
+  generation: 3
   name: test-ytsaurus
   namespace: ytsaurus-components
-  resourceVersion: "22"
+  resourceVersion: "30"
 spec:
   clusterFeatures: {}
   coreImage: ghcr.io/ytsaurus/ytsaurus:stable-25.2.2
@@ -84,7 +84,7 @@
   conditions:
   - lastTransitionTime: null
     message: 0.0.0-devel
-    observedGeneration: 1
+    observedGeneration: 3
     reason: Observed
     status: "True"
     type: OperatorVersion
@@ -102,19 +102,19 @@
     type: defaultMasterInitJobCompleted
   - lastTransitionTime: null
     message: Ready
-    observedGeneration: 1
+    observedGeneration: 3
     reason: Ready
     status: "True"
     type: MasterReady
   - lastTransitionTime: null
     message: 'Pods are ready: 0 of 1, running 0, need 0'
-    observedGeneration: 1
+    observedGeneration: 3
     reason: Ready
     status: "True"
     type: DiscoveryReady
   - lastTransitionTime: null
     message: 'Pods are ready: 0 of 1, running 0, need 0'
-    observedGeneration: 1
+    observedGeneration: 3
     reason: Ready
     status: "True"
     type: HttpProxyReady
@@ -126,23 +126,23 @@
     type: userYtsaurusClientInitJobCompleted
   - lastTransitionTime: null
     message: Cypress operations skipped
-    observedGeneration: 1
+    observedGeneration: 3
     reason: HealthChecks
     status: "True"
     type: UpdateIsPossible
   - lastTransitionTime: null
     message: Ready
-    observedGeneration: 1
+    observedGeneration: 3
     reason: Ready
     status: "True"
     type: YtsaurusClientReady
   - lastTransitionTime: null
     message: Cluster state is Running
-    observedGeneration: 1
+    observedGeneration: 3
     reason: Running
     status: "True"
     type: ReadyToWork
-  observedGeneration: 1
+  observedGeneration: 3
   state: Running
   updateStatus: {}
 
